src/Sensor.cc

```diff
@@ -40,7 +41,7 @@ using namespace sdf;
 
 /// Sensor type strings. These should match the data in
 /// `enum class SensorType` located in Sensor.hh.
-const std::vector<std::string> sensorTypeStrs =
+constexpr std::array<std::string_view, 27> kSensorTypeStrs =
```

Comment:
Can you add a comment here to remind us that we need to update the size template parameter if we add anything to this array.

Reply:
https://github.com/gazebosim/sdformat/pull/1264/commits/b62510fd538f51c6c866ff9a1b477559ddff1e84